src/hotspot/cpu/riscv/stubGenerator_riscv.cpp

```diff
@@ -3045,17 +3040,6 @@ class StubGenerator: public StubCodeGenerator {
       __ leave();
```

Comment:
I think we should use `lookup_secondary_supers_table_const` instead of `lookup_secondary_supers_table`?

Reply:
Done for both ppc and riscv